cf_units/__init__.py

```diff
@@ -175,11 +177,25 @@ def suppress_errors():
         _ud.set_error_message_handler(_default_handler)
 
 
+LOCALE_LOCK = threading.Lock()
+
+
+@contextmanager
+def us_locale():
+    with LOCALE_LOCK:
+        lc_numeric = locale.getlocale(locale.LC_NUMERIC)
+        locale.setlocale(locale.LC_NUMERIC, "en_US.UTF-8")
```

Comment:
Or should I use the "C" locale? I'm not sure if `en_US` is installed on every system.

Reply:
Changed in b1a9302